.travis.yml

```diff
@@ -1,9 +1,14 @@
 sudo: false
 language: cpp
 compiler:
-- clang
+  - clang
 os:
-- osx
+  - osx
+before_install:
+  - wget https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py
```

Comment:
Don't know, there are many things that I don't really like in LLVM's style guide…
For functions, I got used to UpperCamelCase, so lowerCamelCase looks weird, plus I like Google's naming for "cheap" functions (snake_case) that also gives some uniformity with the std style guide.
For enums, I don't see why add prefixes when you can just use a strongly typed enum.